litmus-portal/backend/auth/pkg/utils/invite/templates/template.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html
+    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+</head>
+<body>
+<p>
+    Hello {{.Username}} Here's the link to 
+    <a href="http://localhost:3001/login?#name={{.Username}}&email={{.Password}}" onclick="">Login</a>
```

Comment:
will it be local host for the portal?

Reply:
@Sanjay1611 @gdsoumya @rajdas98 @Saranya-jena

Reply:
Yes this we need to change, it's the link for login page in my case.

Reply:
Yeah IP is required.